src/app/ui/timeline.cpp

```diff
@@ -50,10 +50,18 @@
 #include "she/font.h"
 #include "ui/scroll_helper.h"
 #include "ui/ui.h"
+#include "base/unique_ptr.h"
+#include "doc/algorithm/rotate.h"
+#include "doc/algorithm/resize_image.h"
+#include "she/surface.h"
+#include "she/system.h"
+#include "doc/conversion_she.h"
 
 #include <cstdio>
 #include <vector>
 
+
+
```

Comment:
This is a minor issue, you can remove these lines in case you create other commit for this file.